modules/data-entry/src/main/frontend/src/questionnaire/Questionnaire.jsx

```diff
@@ -0,0 +1,93 @@
+//
+//  Licensed to the Apache Software Foundation (ASF) under one
+//  or more contributor license agreements.  See the NOTICE file
+//  distributed with this work for additional information
+//  regarding copyright ownership.  The ASF licenses this file
+//  to you under the Apache License, Version 2.0 (the
+//  "License"); you may not use this file except in compliance
+//  with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing,
+//  software distributed under the License is distributed on an
+//  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//  KIND, either express or implied.  See the License for the
+//  specific language governing permissions and limitations
+//  under the License.
+//
+
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+
+import { Paper, Grid, Typography, Card, CardHeader, CardContent, withStyles } from "@material-ui/core";
```

Comment:
Our convention for multiple imports is one component per line, in alphabetical order. Reason: It's easier to see is a component is imported or not, updates are easier to read.
```suggestion
import {
  Card,
  CardContent,
  CardHeader,
  Grid,
  Paper,
  Typography,
  withStyles
} from "@material-ui/core";
```